common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -52,7 +52,9 @@ def undo_changes_via_modal(self):
         """
         Trigger clicking event of the undo changes button in the modal.
         Wait for the undoing process to load via ajax call.
+        Before that Scroll so the button is clickable on all browsers
         """
+        self.browser.execute_script("window.scrollTo(0, 500)")
```

Comment:
You also might be able to use scrollIntoView.
See [this stack exchange post](http://stackoverflow.com/questions/3401343/scroll-element-into-view-with-selenium).